### PR TITLE
Cjfirstcss

### PR DESCRIPTION
**Pull request by cbjones93**

### CODE REVIEW

**Review by bvinson23 — CHANGES_REQUESTED** [review]
Everything looks good except I would remove the word "home" off of the Nepal site.

File: html/nepal.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <header class="headerBox">
+        <navbar>
+            <ul class="navList">
+                <li class="navItem"><a href="../index.html">Home</a></li>
+                <li class="navItem"><a href="html/germany.html">Germany</a></li>
+                <li class="navItem"><a href="html/italy.html">Italy</a></li>
+                <li class="navItem active"><a href="html/nepal.html">Nepal</a></li>
+                <li class="navItem"><a href="html/scotland.html">Scotland</a></li>
+            </ul>
+            <section>
```
Comment (bvinson23) [inline html/nepal.html:25]: Not sure what this <section> goes to.

**Review by bvinson23 — APPROVED** [review]
👍 Looks good!

**Review by brandonjon26 — APPROVED** [review]
Lookin good bromigo!

**Pull request merged**